src/DependencyInjection/ProophServiceBusExtension.php

```diff
@@ -118,51 +120,65 @@ private function busLoad(
      */
     private function loadBus(string $type, string $name, array $options, ContainerBuilder $container)
     {
+        $serviceBusName = sprintf('prooph_service_bus.%s', $name);
```

Comment:
String concatenation is used 8 times while `sprintf` is used three times. Perhaps we can unify this to string concatenation?